packages/stark-ui/karma.conf.js

```diff
@@ -8,37 +8,57 @@ const defaultKarmaConfig = require("./node_modules/@nationalbankbelgium/stark-te
 // entry files of the "@nationalbankbelgium/stark-ui" module imported in mock files
 const karmaTypescriptFiles = [{ pattern: helpers.root("index.ts") }, { pattern: helpers.root("public_api.ts") }];
 
-const karmaTypescriptBundlerAliasResolution = {
-	resolve: {
-		alias: {
-			// adapt the resolution of the stark-core module to the UMD module
-			"@nationalbankbelgium/stark-core": "../../dist/packages-dist/stark-core/bundles/stark-core.umd.js",
-			"@nationalbankbelgium/stark-core/testing": "../../dist/packages-dist/stark-core/bundles/stark-core-testing.umd.js",
-			// adapt the resolution of the 3rd party modules used in stark-core
-			"@angularclass/hmr": "../stark-core/node_modules/@angularclass/hmr/dist/index.js",
-			"@ng-idle/core": "../stark-core/node_modules/@ng-idle/core/bundles/core.umd.js",
-			"@ng-idle/keepalive": "../stark-core/node_modules/@ng-idle/keepalive/bundles/keepalive.umd.js",
-			"@ngrx/store": "../stark-core/node_modules/@ngrx/store/bundles/store.umd.js",
-			"@ngrx/effects": "../stark-core/node_modules/@ngrx/effects/bundles/effects.umd.js",
-			"@ngx-translate/core": "../stark-core/node_modules/@ngx-translate/core/bundles/ngx-translate-core.umd.js",
-			"@uirouter/angular": "../stark-core/node_modules/@uirouter/angular/_bundles/ui-router-ng2.js",
-			"@uirouter/core": "../stark-core/node_modules/@uirouter/core/lib/index.js",
-			"@uirouter/rx": "../stark-core/node_modules/@uirouter/rx/lib/index.js",
-			cerialize: "../stark-core/node_modules/cerialize/index.js",
-			"class-validator": "../stark-core/node_modules/class-validator/index.js",
-			"deep-freeze-strict": "../stark-core/node_modules/deep-freeze-strict/index.js",
-			moment: "../stark-core/node_modules/moment/moment.js",
-			ibantools: "../stark-core/node_modules/ibantools/build/ibantools.js"
-		}
-	}
+const karmaTypescriptBundlerAlias = {
+	// adapt the resolution of the stark-core module to the UMD module
+	"@nationalbankbelgium/stark-core": "../../dist/packages-dist/stark-core/bundles/stark-core.umd.js",
+	"@nationalbankbelgium/stark-core/testing": "../../dist/packages-dist/stark-core/bundles/stark-core-testing.umd.js",
```

Comment:
Now, with the "es6-transform" from `karma-typescript` we should point to the FESM or ESM module instead of the UMD ;)

Can you do the same for the other packages in this alias object?

Reply:
I was thinking of doing this with #1145 (amongst other things). But I could append a commit to this PR.

Reply:
Ah ok 😉